### ISSUE TITLE
Web Content Accessibility Guidelines (WCAG) 2.1

### ISSUE THREAD

**Issue by bxo11** [indico/indico#5381]
These guidelines provide information and requirements for Web apps to be more accessible to people with disabilities. Indico needs some corrections/changes to fulfill these requirements.

Example changes:
- Possibility of navigating by keyboard only
  - Increased visibility of focused element (color/frame)
  - No keyboard traps/blocking elements (currently some text editors are blocking focus)
  - Custom order of interactive elements for better keyboard navigation
- Better contrast of all text elements
- Display description for small icons on mouse hover/keyboard focus
- A uniform system of informing about errors/hints/missing elements in the forms
- Confirmation for deleting/changing data
- Possibility to revert applied changes
